securesystemslib/interface.py

```diff
@@ -569,23 +511,15 @@ def import_ed25519_publickey_from_file(filepath):
     An ED25519 key object conformant to
     'securesystemslib.formats.ED25519KEY_SCHEMA'.
   """
-
-  # Does 'filepath' have the correct format?
-  # Ensure the arguments have the appropriate number of objects and object
-  # types, and that all dict keys are properly named.
-  # Raise 'securesystemslib.exceptions.FormatError' if there is a mismatch.
   securesystemslib.formats.PATH_SCHEMA.check_match(filepath)
 
-  # ED25519 key objects are saved in json and metadata format.  Return the
-  # loaded key object in securesystemslib.formats.ED25519KEY_SCHEMA' format that
-  # also includes the keyid.
+  # Load custom on-disk JSON formatted key and convert to its custom in-memory
+  # dict key representation
   ed25519_key_metadata = securesystemslib.util.load_json_file(filepath)
-  ed25519_key, junk = \
-    securesystemslib.keys.format_metadata_to_key(ed25519_key_metadata)
+  ed25519_key, junk = securesystemslib.keys.format_metadata_to_key(
```

Comment:
Nit:  I dislike the use of the `junk` variable name here. I think it's confusing when the accepted Python idiom is to use an `_` variable name for unused return values. By having used a name, I would expect to see the variable used somewhere.

Reply:
Agreed. Will remove.